app/views/News.js

```diff
@@ -70,11 +70,12 @@ class NewsScreen extends Component {
             borderBottomRightRadius: 12,
           }}
           cacheEnabled
-          onLoad={() =>
-            this.setState({
-              visible: false,
-            })
-          }
+          startInLoadingState={true} //Show loading indicator in webweb
```

Comment:
```suggestion
          startInLoadingState
```

Reply:
if startInLoadingState is true, it will show loading indicator on web view. I will re-update comment like "//Show loading indicator on web view".